contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -147,7 +155,6 @@ contract UniswapAnchoredView is UniswapConfig {
             TokenConfig memory config = getTokenConfigBySymbol(symbols[i]);
             string memory symbol = symbols[i];
             bytes32 symbolHash = keccak256(abi.encodePacked(symbol));
-            if (source(messages[i], signatures[i]) != reporter) continue;
```

Comment:
calling attention to this bugfix here. since `i` is an iterator for symbol this is and not `signatures[]`, its wrong. we decided we don't need this change anyway, its ok to try to update the view prices even when the messages passed werent signed by the reporter.